Source/Parallelization/WarpXRegrid.cpp

```diff
@@ -174,6 +174,14 @@ WarpX::RemakeLevel (int lev, Real /*time*/, const BoxArray& ba, const Distributi
     using ablastr::fields::Direction;
     using warpx::fields::FieldType;
 
+    const auto RemakeMultiFab = [&](auto& mf){
```

Comment:
This reintroduces a function that [we removed](https://github.com/ECP-WarpX/WarpX/pull/5230/files#diff-55dfa73c9e66860de4e9da580f1e41bd129ff4e2aab0b7b97e69d26f9adf1b40L172) when introducing the `MultiFabRegister` (#5230). 

It could be removed once we introduce an `iMultiFabRegister`.